admin-dev/themes/new-theme/js/pages/order/create/create-order-map.js

```diff
@@ -74,6 +75,7 @@ export default {
   orderPaymentMethod: '.js-order-payment-method',
   orderStatusField: '.js-order-status',
   emptyListRowTemplate: '#js-empty-list-row',
+  loadingListRowTemplate: '#js-loading-list-row',
```

Comment:
I think a JS selector should never be an ID as it should be reusable :/

What do you think ?

Reply:
indeed if template will be used multiple times, it will generate html with duplicated ids, which will issue warnings. Same mistake was done in order view page i guess (see screenshot).
![Screenshot from 2020-03-27 11-08-06](https://user-images.githubusercontent.com/31609858/77740103-5292c600-701b-11ea-955d-025e2f7fcc1d.png)

Reply:
This one is a bit special, it is used to get the template that is cloned It maps a `script` tag, and we only duplicate its content (which has no ids, only classes):
`const $emptyTableRow = $($(createOrderMap.loadingListRowTemplate).html()).clone();`

I could use a class to identify this script but it wouldn't make much sense because it really is not a component, more like a component resource at worst And it's probably better to identify it via an id to emphasize its difference

Now the whole script/template may not be a good practice may not be the best, but it's been used a lot in all pages so it's not the purpose o this PR to refacto all this

Reply:
I guess its the case
> On the other hand, if it only stores the template inside and the id is not rendered multiple times, then it might be the best option.

So it seems all good to me